README.md

```diff
@@ -44,41 +47,52 @@ Read more in the whitepaper: https://whitepaper.fission.codes/access-control/uca
  `prf`, Proof, an optional nested token with equal or greater privileges.
 
  ### Signature
+
  A signature (using `alg`) of the base64 encoded header and payload concatenated together and delimited by `.`
 
-## Build params
-Use `ucan.build` to help in formatting and signing a ucan. It takes the following parameters
+
+
+## Build
+
+`ucan.build` can be used to help in formatting and signing a UCAN. It takes the following parameters:
 ```ts
-export type BuildParams = {
-  // to/from
-  audience: string
+type BuildParams = {
+  // from/to
   issuer: Keypair
+  audience: string
 
   // capabilities
-  capabilities: Array<Capability>
+  capabilities?: Array<Capability>
 
   // time bounds
   lifetimeInSeconds?: number // expiration overrides lifetimeInSeconds
   expiration?: number
   notBefore?: number
 
-  // proof / other info
+  // proofs / other info
   facts?: Array<Fact>
-  proof?: string
-
-  // in the weeds
-  ucanVersion?: string
+  proofs?: Array<string>
+  addNonce?: boolean
 }
 ```
+
 ### Capabilities
-`capabilities` is an array of resources and permission level formatted as:
+
+`capabilities` is an array of resource pointers and abilities:
 ```ts
 {
-  $TYPE: $IDENTIFIER,
-  "cap": $CAPABILITY
+  // `with` is a resource pointer in the form of a URI, which has a `scheme` and `hierPart`.
+  // → "mailto:boris@fission.codes"
+  with: { scheme: "mailto", hierPart: "boris@fission.codes" },
+
+  // `can` is an ability, which always has a namespace and optional parts.
```

Comment:
Might be worth changing "optional parts" to "optional segments" to match the `segments` param.